src/service-override/editor.ts

```diff
@@ -98,5 +98,6 @@ export {
   OpenEditor,
   IEditorOptions,
   IResolvedTextEditorModel,
-  IReference
+  IReference,
+  EditorService
```

Comment:
If it's exposed, we should probably give it a less generic name, like `SimpleEditorService` or something, don't you think?

Reply:
Good one, will do!

Reply:
Just change the commit message and we're good to go :)